locales/en/index.yml

```diff
@@ -2873,10 +2873,17 @@ features:
         title: "Attachments"
         toast:
           success: "Download complete"
-          failure: "An error occurred while downloading the attachment"
         bottomSheet:
-          title: "Pay attention to attachments"
-          body: "IO cannot guarantee the security of the attachment - save to your device or share only the files that you think are safe enough."
+          warning:
+            title: "Pay attention to attachments"
+            body: "IO cannot guarantee the security of the attachment - save to your device or share only the files that you think are safe enough."
+          loading:
+            title: "Attachment"
+            body: "Downloading the attachment"
+          failing:
+            title: "Attachment"
+            body: "Non sono riuscito a scaricare l'allegato"
+            details: "Per favore, controlla la tua connessione e riprova"
```

Comment:
perhaps they should be in english 😄

Reply:
Definitely! Thanks! https://github.com/pagopa/io-app/pull/3706/commits/cb9b2d7f97f0cda8a3727995a56317dd81e9e2b1